Directory.Packages.props

```diff
@@ -79,4 +79,7 @@
     <PackageVersion Include="Microsoft.VariantAssignment.Client" Version="2.4.17140001" />
     <PackageVersion Include="Microsoft.VariantAssignment.Contract" Version="3.0.16990001" />
   </ItemGroup>
+  <ItemGroup>
+    <GlobalPackageReference Include="Microsoft.Build.CopyOnWrite" Version="1.0.282" />
+  </ItemGroup>
```

Comment:
Given the addition to Dir.Build.targets you should not need this

Reply:
My first attempt was with only this change on Directory.Build.props and nothing else... the build failed and I assumed it was because of the .vcxproj files here so I added both!

I will remove this one and see what happens!